core/src/operations/tests.rs

```diff
@@ -0,0 +1,12 @@
+#[cfg(test)]
+mod tests {
```

Comment:
You don't need to wrap this in a `tests` module any more because the whole file is a `tests` module. With the current structure we actually get `tests::tests` for the module.